src/functionaltest/java/com/ericsson/ei/frontend/pageobjects/SubscriptionPage.java

```diff
@@ -192,4 +191,8 @@ public void clickReloadLDAP(String response, String responseAuth) throws IOExcep
                 .until(ExpectedConditions.elementToBeClickable(By.id("reloadButton")));
         reloadBtn.click();
     }
+
+    public boolean textExistsInTable(String loc) {
```

Comment:
Do we really need this method? I think you can use buttonExist method where you invoking textExistsInTable